c_tests/run_trace_compiler.rs

```diff
@@ -0,0 +1,57 @@
+//! Lang tester harness for trace compiler tests.
+
+#![feature(once_cell)]
+
+use lang_tester::LangTester;
+use regex::Regex;
+use std::{env, fs::read_to_string, path::PathBuf, process::Command};
+
+const COMMENT: &str = ";";
+
+fn main() {
+    println!("Running trace compiler tests...");
```

Comment:
I don't think this is useful -- it'll just get in the way of `cargo test` style output I think?

Reply:
I think it's useful to see what suite a failure is coming from. The problem is that lang test tests always identify as `lang_tests::` regardless of which suite they came from:

```
Running C tests with -Oz...
...
test lang_tests::simple_interp_loop1.c ... ok                                                              
test lang_tests::simple_interp_loop2.c ... ok
                                                                                                           
test result: ok. 44 passed; 0 failed; 31 ignored; 0 measured; 0 filtered out                               
                                                     
     Running run_trace_compiler.rs (target/debug/deps/trace_compiler_tests-2ecdc66af5638631)               
Running trace compiler tests...          
                                                     
running 4 tests                                                                                            
test lang_tests::smallest.ll ... ok           
test lang_tests::two_blocks.ll ... ok                 
```

As you can see already have similar prints for the C tests (where otherwise you wouldn't know what optimisation level a failure came from), and I don't think the prints look out of place.

Maybe this is something that could be addressed in `lang_tester`. If the suite name could be customised, we could see:

```
test c_prog_O0::my_test... ok
test c_prog_Oz::my_test... ok
test trace_compiler::other_test... fail
```

Then we wouldn't need extra prints.